### DISCUSSION TITLE
[question] How do you create Charts from NocoBb data ?

### DISCUSSION THREAD

**Discussion by Cyrille37** [nocodb/nocodb#317] [Q&A]
Hello
This NocoDb makes me wriggling :-) So nice <3

There one thing I did not find in demos: **Charts**...
Is there some stuff about in the roadmap, or do you use another tool connected to NocoDb ?

Thanks & cheers

**Comment by o1lab** [comment]
Thank you.

For the moment there is no chart feature. 

Can think of it coming as a plugin.

**Comment by jopellerin** [comment]
I came across this discussion looking for charts for my nocodb data.
I ended up using Metabase to display my data in all sorts of ways, including charts. Just connect it to the nocodb database and you're in business. They also have a lot of documentation on how to do stuff.

I thought this might be helpful to others.

**Reply by o1lab** [reply]
+ +superset can also do

**Reply by almereyda** [reply]
Or Redash

**Reply by LucBerge** [reply]
@o1lab @almereyda

- **Self-hosted** : If I use Superset or Redash to connect to the underlying MySQL database, the tables and fields does not correspond to what is visible on Nocodb. This is expected since it has all the table schema, fields type ...etc Then how do you correctly plot charts ?
- **Cloud version** : How do you connect to Cloud version ?

**Comment by lpil** [comment]
Is there a way to do this?

**Comment by LucBerge** [comment]
Hello,
not having charts embedded in nocodb is a red flag for me. I need KPI computed from my data otherwise it is useless.
The only bypass is to connect a chart tool to compute my KPIs.

**Comment by LucBerge** [comment]
Hello all,
If you want to push this feature, there is [an open issue here](https://github.com/nocodb/nocodb/issues/9223). Upvote and leave a comment to make it visible and accelerate the process.

**Reply by clesecq** [reply]
Upvote, of course. But leaving a comment will notify all those following the issue.

**Reply by LucBerge** [reply]
@StelFux including the developers
